### PR TITLE
Add python-cityhash

### PR DESCRIPTION
**Pull request by charlesbluca**
Attempting to supersede https://github.com/conda-forge/staged-recipes/pull/11057 since there's some renewed interest in getting this packaged from Dask (xref https://github.com/dask/dask/pull/10161#discussion_r1163326263)

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by charlesbluca — COMMENTED** [review]
Looks like this package is also on `bioconda` under `cityhash`:

- https://github.com/bioconda/bioconda-recipes/blob/master/recipes/cityhash/meta.yaml

Should we still open up a discussion post to remove the package from that channel?

File: recipes/python-cityhash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  home: https://github.com/escherba/python-cityhash
+  summary: Python bindings for CityHash and FarmHash
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - charlesbluca
+    - escherba
+    - jrbourbeau
```
[the remark this answers is not in the document]
Comment (jrbourbeau) [reply recipes/python-cityhash/meta.yaml:49]: Thanks @charlesbluca! But I'd prefer to not be included in this one

File: recipes/python-cityhash/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/cityhash/cityhash-{{ version }}.tar.gz
+  sha256: 087c1c9f3d0ca3303bbcf683295151304274059d1efba07ee8429b3b80c33af0
+
+build:
+  skip: true  # [py2k]
```
Comment (dopplershift) [inline recipes/python-cityhash/meta.yaml:13]: conda-forge isn't building Python 2 any more.
```suggestion
```

File: recipes/python-cityhash/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (charlesbluca) [reply recipes/python-cityhash/meta.yaml:49]:
```suggestion
    - escherba
```

**Comment by dopplershift** [comment]
@charlesbluca If you would like the package included in conda-forge, it does need to be removed from bioconda.

**Comment by charlesbluca** [comment]
Sounds good to me, have opened up a discussion post around this and a PR removing it:

- https://github.com/orgs/conda-forge/teams/bioconda-recipes/discussions/16
- https://github.com/bioconda/bioconda-recipes/pull/40418

**Review by bgruening — APPROVED** [review]
I will merge the bioconda PR.

File: recipes/python-cityhash/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+about:
+  home: https://github.com/escherba/python-cityhash
+  summary: Python bindings for CityHash and FarmHash
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - charlesbluca
+    - escherba
```
Comment (charlesbluca) [inline recipes/python-cityhash/meta.yaml:47]: Going to go ahead and remove for now to avoid potentially unwanted notifications


```suggestion
```

**Pull request merged**